### PR TITLE
Jr comments likes follow requests

### PR DESCRIPTION
**Pull request by jb520**
Added javascript to improve comment, like, and follow/unfollow actions.

### CODE REVIEW

File: app/views/follow_requests/_follow_unfollow.html.erb [hunk 1]
```diff
@@ -1,17 +1,19 @@
-<% unless sender == recipient %>
-  <% follow_request = current_user.sent_follow_requests.find_by(recipient: recipient) %>
+<div class="<%= dom_id(sender) %>_<%= dom_id(recipient) %>_follow_unfollow">
```
Comment (MayaS1111) [inline app/views/follow_requests/_follow_unfollow.html.erb:1]: Good job for having both the sender and recipient and using "class"

**Review by MayaS1111 — APPROVED** [review]
Looks great!

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -59,6 +62,8 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Comment was successfully destroyed." }
       format.json { head :no_content }
+
```
Comment (ahuynh3a) [inline app/controllers/comments_controller.rb:65]: great job remembering to add format.js to your methods, this is a nitpik, but remember to be mindful of white spaces, removing any unnecessary space

File: app/views/comments/_comment.html.erb [hunk 3]
```diff
@@ -11,11 +11,11 @@
     </div>
     <div>
       <% if current_user == comment.author %>
-        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted", remote: true do %>
```
Comment (ahuynh3a) [inline app/views/comments/_comment.html.erb:14]: Great job being mindful of when to use remote: true and local: false

File: app/views/comments/edit.js.erb [hunk 4]
```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@comment) %>").replaceWith("<%= j(render "comments/form", comment: @comment) %>");
```
Comment (ahuynh3a) [inline app/views/comments/edit.js.erb:1]: This is great! You can use variables to make your code clearer to read as well!

File: app/views/follow_requests/_follow_unfollow.html.erb [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
-    <% if follow_request.pending? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
-        Un-request
-      <% end %>
-    <% elsif follow_request.accepted? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
-        Un-follow
+    <% if follow_request %>
+      <% if follow_request.pending? %>
+        <%= link_to follow_request, method: :delete, class: "btn btn-outline-secondary", remote: true do %>
```
Comment (ahuynh3a) [inline app/views/follow_requests/_follow_unfollow.html.erb:7]: great job switching to remote: true

File: app/views/follow_requests/create.js.erb [hunk 6]
```diff
@@ -0,0 +1,3 @@
+console.log("i clicked follow")
```
Comment (ahuynh3a) [inline app/views/follow_requests/create.js.erb:1]: haha love this log

File: app/views/follow_requests/create.js.erb [hunk 7]
```diff
@@ -0,0 +1,3 @@
...
+$(".<%= dom_id(@follow_request.sender) %>_<%= dom_id(@follow_request.recipient)%>_follow_unfollow").replaceWith("<%=j(render 'follow_requests/follow_unfollow', sender: @follow_request.sender, recipient: @follow_request.recipient)%>");
```
Comment (ahuynh3a) [inline app/views/follow_requests/create.js.erb:2]: this is great! remember to assign your jquery to variables so that your code is clearer to read!

**Review by ahuynh3a — APPROVED** [review]
Great Job Jennifer! Your code is clean and easy to read! I just had some nitpik comments, but overall it was fantastic!

**Pull request merged**